misc/cspell.json

```diff
@@ -49,6 +50,7 @@
     "markdownlint",
     "mempool",
     "Merkelized",
+	   "merklized",
```

Comment:
Is this spelling used in text? I thought we use "Merkelized" (which is mentioned directly above 🙃).

Reply:
deleted